packages/desktop-gui/cypress/integration/specs_list_spec.coffee

```diff
@@ -238,7 +238,14 @@ describe "Specs List", ->
         it "shows empty message if no results", ->
           cy.get(".filter").clear().type("foobarbaz")
           cy.get(".outer-files-container").should("not.exist")
-          cy.get(".empty-well").should("have.text", "No files match the filter 'foobarbaz'")
+          cy.get(".empty-well").should("have.text", 'No specs match your search: "foobarbaz"')
+
+        it.only "clears and focuses the filter field when clear search is clicked", ->
```

Comment:
how did `.only` get there! don't we have http://github.com/bahmutov/stop-only set up?!

Reply:
Not sure, it didn't flag for me. Fixed though

Reply:
![image](https://user-images.githubusercontent.com/1151760/51269988-cfdf9700-1991-11e9-80ce-6e72bfade111.png)

Reply:
It's only searching `.js` files and missing all the `.coffee` files